js/filter-panel.js

```diff
@@ -25,21 +28,25 @@ function FilterPanel(options) {
   this.$toggle.on('click', this.toggle.bind(this));
 
   this.filterSet = new FilterSet(this.$form).activate();
-  if (!_.isEmpty(this.filterSet.serialize())) {
-    this.show();
-  }
 
   this.adjust();
 }
 
 FilterPanel.prototype.adjust = function() {
-  if ($('body').width() > 768) {
+  if ($(document).width() > helpers.BREAKPOINTS.LARGE) {
```

Comment:
Given that this line is repeated in the `setHeight` method and may change again in the future, would it make sense to move this conditional into its own method, or do you think that's getting too granular?